common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -12,32 +12,30 @@
 }
 """
 
-import pymongo
-import sys
-import logging
 import copy
+from datetime import datetime
+from importlib import import_module
+import logging
+import pymongo
 import re
+import sys
 from uuid import uuid4
 
 from bson.son import SON
-from datetime import datetime
+from contracts import contract, new_contract
 from fs.osfs import OSFS
 from mongodb_proxy import autoretry_read
-from path import Path as path
-from pytz import UTC
-from contracts import contract, new_contract
-
-from importlib import import_module
 from opaque_keys.edx.keys import UsageKey, CourseKey, AssetKey
-from opaque_keys.edx.locations import Location, BlockUsageLocator
-from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from opaque_keys.edx.locations import Location, BlockUsageLocator, SlashSeparatedCourseKey
 from opaque_keys.edx.locator import CourseLocator, LibraryLocator
-
+from path import Path as path
+from pytz import UTC
 from xblock.core import XBlock
 from xblock.exceptions import InvalidScopeError
 from xblock.fields import Scope, ScopeIds, Reference, ReferenceList, ReferenceValueDict
 from xblock.runtime import KvsFieldData
 
+from lms.djangoapps.courseware.field_overrides import OverrideModulestoreFieldData
```

Comment:
Please don't introduce dependencies from common -> lms. Instead, prefer passing in the `FieldData` wrapping functions into the `Modulestore` in `__init__`.